src/tmx/Asn_J2735/src/r63/TestMessage01.c

```diff
@@ -40,7 +39,7 @@ asn_SEQUENCE_specifics_t asn_SPC_TestMessage01_specs_1 = {
 	offsetof(struct TestMessage01, _asn_ctx),
 	asn_MAP_TestMessage01_tag2el_1,
 	2,	/* Count of tags in the map */
-	asn_MAP_TestMessage01_oms_1,	/* Optional members */
+	0, 0, 0,	/* Optional elements (not needed) */
```

Comment:
@dan-du-car  Can you explain why these were included in the first place and why they are no longer necessary? Is this just a ASN1 schema difference that results in different generated code? If so do we know the difference

Reply:
see the above